temoa/cli.py

```
@@ -437,6 +439,165 @@ def migrate(
+    except (ModuleNotFoundError, FileNotFoundError, AttributeError) as e:
+        logger.exception('Failed to load tutorial resources from package')
+        # Fallback to development paths (for development environments)
+        fallback_config = Path(__file__).parent / 'tutorial_assets' / 'config_sample.toml'
+        fallback_database = Path(__file__).parent / 'tutorial_assets' / 'utopia.sqlite'
+
+        if not fallback_config.exists() or not fallback_database.exists():
+            raise FileNotFoundError(
+                f'Tutorial resources not found. Tried package resources and fallback paths:\n'
+                f'Config: {fallback_config}\n'
+                f'Database: {fallback_database}'
+            ) from e
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)